docs/end-user/annotations.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Annotations
```

Comment:
```suggestion
title: Service Account Integration
```

Reply:
Annotation may be not clear enough for this feature and users can hardly find the docs.

I think we can just clarify the service account mechanism here and tell users how to use it. Give some examples.

Reply:
Thanks for the feedback. I added an example and background to the document. Could you re-review?